src/components/Filter/Filter.style.ts

```diff
@@ -50,7 +50,7 @@ export const buttonStyle = ({
       activeCalculatedColor,
       calculatedColor,
     }),
-    boxShadow: styleType === 'elevated' && !hasSelectedValue ? theme.elevation['01'] : 'none',
+    boxShadow: styleType === 'elevated' ? theme.elevation['01'] : 'none',
```

Comment:
For consistency reasons as well as readability I would suggest to follow a similar pattern to `wrapperStyleSwitch` that's defined in `src/components/AssetMatching/AssetMatching.style.ts`  when for properties based on styleType.

Reply:
Excuse my delayed reply! Yes that is an absolutely nice suggestion! thank you! I really tried to simplify the css of this component but didn't manage to provide a "clean" solution. The style of the wrapper is not relevant only with the `styleType` but also if the the menu is open and if there is and selectedItem, so a switch case won't change a lot these parts. However if you have any specific suggestions on that i would be more than grateful if you share with me.

Reply:
Well , I was only talking about the boxShadow prop that was visible in the diff. After checking out more the code, I can see what you are talking about.